src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -159,7 +159,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                             "${qnIdx}", Integer.toString(qnIdx),
                             "${responseIdx}", Integer.toString(responseIdx),
                             "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled=\"disabled\"",
+                            "${disabled}", sessionIsOpen ? "" : "disabled=\"\"",
```

Comment:
It's the same, but on `HtmlHelper` and `ElementTag` the attribute-value pair is enforced (you can't build `disabled` or alike on its own), so we might as well standardize it.